webui/react/src/pages/Settings/SettingsAccount.tsx

```diff
@@ -45,7 +44,7 @@ const SettingsAccount: React.FC = () => {
         updateUser(user.id, (oldUser) => ({ ...oldUser, displayName: newValue }));
         message.success(API_DISPLAYNAME_SUCCESS_MESSAGE);
       } catch (e) {
-        message.error(API_DISPLAYNAME_ERROR_MESSAGE);
+        message.error((e as DetError).message);
         handleError(e, { silent: true, type: ErrorType.Input });
```

Comment:
nit: I think this should pop up an error message if it's not silenced

Reply:
Right! Thanks!